src/marine/smap_sss2ioda.py

```diff
@@ -25,7 +25,7 @@
 from orddicts import DefaultOrderedDict
 
 
-vName = "sea_surface_salinity"
+vName = "salinity"
```

Comment:
@gthompsnJCSDA @guillaumevernieres double checking this is correct. How will UFO know this is 'sea surface' now? Does a pressure/height MetaData variable need added?